src/libraries/System.Net.Primitives/src/System/Net/IPEndPoint.cs

```diff
@@ -62,6 +64,7 @@ public IPEndPoint(IPAddress address, int port)
         /// <summary>
         /// Gets or sets the IP address.
         /// </summary>
+        [AllowNull]
```

Comment:
The setter throws if `value` is null, so probably best to remove this attribute.

Reply:
Fixed.